client/client-kv.c

```diff
@@ -0,0 +1,240 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <netdb.h>
+#include <unistd.h>
+#include <inttypes.h>
+#include <string.h>
+#include "../lib/marshal/marshal.h"
+/****************************************
+ Author: Joel Klein, Katie Stasaski, Lucas Chaufournier, Tim Wood
+ with a little help from
+ http://beej.us/guide/bgnet/
+ ****************************************/
+//Returns the number of bytes to wait for
+
+struct operation* read_line(int sockfd){
```

Comment:
For all following function prototypes, just be sure to adhere to the style guide